### DISCUSSION TITLE
Biological meaning of peak-gene link with negative score

### DISCUSSION THREAD

**Discussion by yamajackr** [stuart-lab/signac#1301] [Q&A]
Hi @timoast 
Thank you again for the great tool. I have a question about the biological meaning of peak-gene links with negative scores.
Some researchers assumed that activators increase the accessibility of their target regions and repressors decrease the accessibility and then regulate the target gene expression (DOI: [10.1016/j.celrep.2019.10.106](https://doi.org/10.1016/j.celrep.2019.10.106)). In other words, they assumed that the cis-regulatory regions are positively correlated with the associated gene expression. In your paper, 80% of the linked regions had a positive correlation, and the association between frequency and the distance to TSS seemed reasonable. In contrast, links with negative coefficients didn't seem to be affected by the distance to TSS. So I guessed many links with negative scores were false positive links.
What do you think of the links with negative scores in a biological meaning? Should we discard the links for downstream analysis?
I'd be happy if I could hear an expert opinion.

Best, 
Ryosuke

**Comment by aarebeca** [comment]
Hello, 
Have you got any answer regarding the meaning of negative scores?
Thanks!
